manifest.json

```diff
@@ -127,6 +127,30 @@
       "headers": [
         "entry"
       ]
+    },
+    {
+      "url": "https://global-sublime-static-lists.s3.amazonaws.com/latest/abuse_ch_malwarebazaar_sha256_trusted_reporters.txt",
+      "identifier": "abuse_ch_malwarebazaar_sha256_trusted_reporters",
+      "display_name": "Abuse.ch: MalwareBazaar SHA256 hashes. Trusted reporters: pr0xylife, abuse_ch, cocaman, James_inthe_box, JAMESWT_MHT",
+      "headers": [
+        "entry"
+      ]
+    },
+    {
+      "url": "https://global-sublime-static-lists.s3.amazonaws.com/latest/abuse_ch_urlhaus_urls_trusted_reporters.txt",
+      "identifier": "abuse_ch_urlhaus_urls_trusted_reporters",
+      "display_name": "Abuse.ch: URLhaus URLs. Trusted reporters: Cryptolaemus1, abuse_ch",
+      "headers": [
+        "entry"
+      ]
+    },
+    {
+      "url": "https://global-sublime-static-lists.s3.amazonaws.com/latest/abuse_ch_urlhaus_domains_trusted_reporters.txt",
+      "identifier": "abuse_ch_urlhaus_domains_trusted_reporters",
+      "display_name": "Abuse.ch: URLhaus domains derived from trusted reporter URLs. Trusted reporters: Cryptolaemus1, abuse_ch",
```

Comment:
> derived from trusted reporter URLs

confusing. They are derived from URL lists and filtered by trusted reporters. This sounds like "trusted reporter URLs" is a thing

Reply:
how about `URLs derived from $abuse_ch_urlhaus_urls_trusted_reporters`

Reply:
Minor tweak to: `Domains derived from $abuse_ch_urlhaus_urls_trusted_reporters`